### PR TITLE
DOCS: Add EDB configuration example.

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: examples/edb_cfg_examples/setup_edb_for_signal_integrity_analysis.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+# All project files are saved in the folder ``temp_file.dir``. If you've run this example as a Jupyter notbook you
+# can retrieve those project files. The following cell removes all temporary files, including the project folder.
+
+# ## Cleanup
+#
+# All project files are saved in the folder ``temp_file.dir``. If you've run this example as a Jupyter notbook you
+# can retrieve those project files. The following cell removes all temporary files, including the project folder.
+
+time.sleep(3)
```
Comment (SMoraisAnsys) [inline examples/edb_cfg_examples/setup_edb_for_signal_integrity_analysis.py:185]: Not sure this is required anymore. There was an issue with the desktop close call but I think it got fixed recently.

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

File: examples/edb_cfg_examples/setup_edb_for_signal_integrity_analysis.py (hunk 1 above)
Comment (ring630) [reply examples/edb_cfg_examples/setup_edb_for_signal_integrity_analysis.py]: Then, I will remove it.

**Comment by SMoraisAnsys** [comment]
Lets put this PR as a draft while I'm switching our documentation

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Pull request merged**